65.md

```diff
@@ -8,57 +8,114 @@ Relay List Metadata
 
 Defines a replaceable event using `kind:10002` to advertise preferred relays for discovering a user's content and receiving fresh content from others.
 
-The event MUST include a list of `r` tags with relay URIs and a `read` or `write` marker. Relays marked as `read` / `write` are called READ / WRITE relays, respectively. If the marker is omitted, the relay is used for both purposes.
+The event SHOULD include `relay-usage` tags, each with a relay URIs and a string indicating what the relay is used for. They should also include tags for backwards compatibility (see section below).
+
+`relay-usage` tags are in the following form:
+
+```json
+["relay-usage", "<relay-url>", "<usage-flags>"]
```

Comment:
Imo it's better to use `ru` instead of `relay-usage` as tag key to make the event smaller (we should strive to save space cause relays aren't supposed to delete them often). Two letters is enough for relays to skip indexing.

Reply:
Sounds good. The Russians will love it ;-)